src/tests/fuzz/fuzz_sigpcap_aware.c

```diff
@@ -181,6 +182,17 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
         p->pcap_cnt = pcap_cnt;
     }
     PacketFree(p);
+    for (uint32_t u = 0; u < flow_config.hash_size; u++) {
```

Comment:
can this be using FlowShutdown or can that be refactored to share code? I don't like seeing this loop multiple times.